bika/lims/locales/es/LC_MESSAGES/bika.po

```diff
@@ -3666,11 +4024,18 @@ msgid "No sampling workflow"
 msgstr ""
 
 #: bika/lims/browser/templates/login_details.pt:141
-msgid "No user exists for ${contact_fullname} and he/she will not be able to log in. Fill in the form below to create one for him/her."
-msgstr "No hay datos de acceso registrados para el contacto ${contact_fullname}, por lo que no podrá acceder al sistema. Puede crear las credenciales de acceso rellenando el formulario siguiente."
+msgid ""
+"No user exists for ${contact_fullname} and he/she will not be able to log "
+"in. Fill in the form below to create one for him/her."
+msgstr ""
+"No hay datos de acceso registrados para el contacto ${contact_fullname}, por "
+"lo que no podrá acceder al sistema. Puede crear las credenciales de acceso "
+"rellenando el formulario siguiente."
```

Comment:
@xispa just a matter of style but, what about _rellenando el siguiente formulario_?